AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/GraphQLWithUserPoolIntegrationTests-amplifyconfiguration.json

```diff
@@ -0,0 +1,16 @@
+{
+    "UserAgent": "aws-amplify-cli/2.0",
+    "Version": "1.0",
+    "api": {
+        "plugins": {
+            "awsAPIPlugin": {
+                "graphqlwithuserpooli": {
+                    "endpointType": "GraphQL",
+                    "endpoint": "https://b6i3uk4txjaezmhgp7p5st4jnu.appsync-api.us-west-2.amazonaws.com/graphql",
```

Comment:
As above, this should not have been committed with real content. Obfuscate on next commit and remove the project that generated it.

Reply:
done